rdkit/Chem/Draw/__init__.py

```diff
@@ -427,6 +428,230 @@ def ReactionToImage(rxn, subImgSize=(200,200),**kwargs):
     res.paste(nimg,(i*subImgSize[0],0))
   return res
 
+def is2D(conf):
+  """returns True if a conf has 2d coords"""
+  allZeroes = True
+  for atompos in range(conf.GetNumAtoms()):
+    x,y,z = conf.GetAtomPosition(atompos)
+    if z != 0.0:
+      return False
+    if x != 0.0 or y != 0.0:
+      allZeroes = False
+  return not allZeroes
+
+def _cleanupRXN(rxn):
+  """Cleanup reactions so the computed coords are sane for depiction"""
+  for mol in rxn.GetReactants():
+    compute2D=False
+    for conf in mol.GetConformers():
+      compute2D = not is2D(conf)
+
+    try:
+      AllChem.SanitizeMol(mol, 
+                          sanitizeOps=(AllChem.SanitizeFlags.SANITIZE_ALL^ 
+                                       AllChem.SanitizeFlags.SANITIZE_KEKULIZE))
+    except:
+      pass
+    if compute2D:
+      AllChem.Compute2DCoords(mol)
+
+  for mol in rxn.GetProducts():
+    compute2D=False
+    for conf in mol.GetConformers():
+      compute2D = not is2D(conf)
+    try:
+      AllChem.SanitizeMol(mol, 
+                          sanitizeOps=(AllChem.SanitizeFlags.SANITIZE_ALL^ 
+                                       AllChem.SanitizeFlags.SANITIZE_KEKULIZE))
+    except:
+      pass
+    if compute2D:
+      AllChem.Compute2DCoords(mol)  
+  return rxn
+
+#source https://commons.wikimedia.org/wiki/File:Tab_plus.svg
+#  creative commons license (need attribution?  Is link enough? Make our own?)
+
+svg_plus = '''<path d="M -5,95 L 25.950102,95" style="opacity:0.5;fill:#000000;fill-opacity:1 " id="p1" />
+<path d="M -5.4181532,95 L 25,95" style="opacity:0.5;fill:#000000;fill-opacity:0.5" id="p2" />
+<text x="6.9755859" y="14.5" style="font-size:10px;font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;text-align:start;line-height:125%;writing-mode:lr-tb;text-anchor:start;opacity:1;fill:#000000;fill-opacity:1;stroke:#000000;stroke-width:1;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:4;stroke-dasharray:none;stroke-dashoffset:0;stroke-opacity:1;font-family:Arial" id="t" xml:space="preserve">
```

Comment:
`font-family` should be `sans-serif` to be more generic and match the rest of the SVG.